flake.nix

```diff
@@ -0,0 +1,34 @@
+{
+  description = "Future-proof note-taking and publishing based on Zettelkasten";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs";
+    flake-utils.url = "github:numtide/flake-utils";
+    flake-compat = {
+      url = "github:edolstra/flake-compat";
+      flake = false;
+    };
+  };
+
+  outputs = { self, nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+
+        project = import ./project.nix { inherit pkgs; };
+
+        neuron = project.neuron;
+        neuronApp = flake-utils.lib.mkApp { drv = neuron; };
+
+        shell = project.shell;
```

Comment:
remove `neuron` and `shell` variables.

I think you might be able to get away with removing the `neuronApp` variable as well, and do something like this further below:

```nix
rec {
  apps = { neuron = flake-utils.lib.mkApp { drv = project.neuron; };
  defaultApp = apps.neuron;
}
```

(note the `rec`).